docs/plugins/examples.rst

```diff
@@ -53,7 +53,7 @@ Enable SAML authentication
 
       openedx-lms-common-settings: |
         # saml special settings
-        AUTHENTICATION_BACKENDS += ["third_party_auth.saml.SAMLAuthBackend", "django.contrib.auth.backends.ModelBackend"]
+        THIRD_PARTY_AUTH_BACKENDS = ["third_party_auth.saml.SAMLAuthBackend"]
```

Comment:
Are you quite sure that this is the right fix? Looking at the source code, the `THIRD_PARTY_AUTH_BACKENDS` setting is not read at all. I believe that the import error is caused by the fact that we should now specify the full import path: `common.djangoapps.third_party_auth.saml.SAMLAuthBackend`.

Can you please test again your plugin and my proposed change, see which one works?